Tests/Fixtures/Generated/Strings/structured-swift5/localizable-customName.swift

```diff
@@ -8,7 +8,7 @@ import Foundation
 // MARK: - Strings
 
 // swiftlint:disable explicit_type_interface function_parameter_count identifier_name line_length
-// swiftlint:disable nesting type_body_length type_name
+// swiftlint:disable nesting type_body_length type_name vertical_whitespace_opening_braces
```

Comment:
Not sure why this violation exception is now needed, what changed?

Reply:
I added that exception a year ago because I was getting a swiftlint warning in every generated file. I just reverted it and got no warnings. Must have been an issue with swiftlint